src/components/executive-education-2u/ExecutiveEducation2UError.jsx

```diff
@@ -29,7 +30,17 @@ const ExecutiveEducation2UError = ({ failureReason, httpReferrer }) => {
             Please contact your edX administrator to resolve the error and gain access to this content.
           </p>
           {httpReferrer && (
-            <Button href={httpReferrer} iconBefore={ArrowBack} variant="primary">
+            <Button
+              href={httpReferrer}
+              iconBefore={ArrowBack}
+              variant="primary"
+              onClick={() => {
+                sendEnterpriseTrackEvent(
```

Comment:
Because this link currently doesn't open in a new tab/window, there is a chance (depending on network connection, etc) that the user's browser navigates to the `httpReferrer` before the API call to Segment has resolved.

Either we should make this button open in a new window, or we may need to use the "hack" of intercepting the default click behavior to add a 300ms delay (which is what Segment's `trackLink` function does under the hood) to give ample time for the analytics track event to resolve while not introducing a perceivable delay.